server/asset/dcr/dcr.go

```diff
@@ -628,6 +631,74 @@ func (dcr *Backend) VerifyUnspentCoin(ctx context.Context, coinID []byte) error
 	return nil
 }
 
+// BondVer returns the latest supported bond version.
+func (dcr *Backend) BondVer() uint16 {
+	return BondVersion
+}
+
+// ParseBondTx makes the package-level ParseBondTx pure function accessible via
+// a Backend instance. This performs basic validation of a serialized
+// time-locked fidelity bond transaction given the bond's P2SH redeem script.
+func (*Backend) ParseBondTx(ver uint16, rawTx []byte) (bondCoinID []byte, amt int64, bondAddr string,
+	bondPubKeyHash []byte, lockTime int64, acct account.AccountID, err error) {
+	return ParseBondTx(ver, rawTx)
```

Comment:
Pure functions are more usable in general, but for server/dex, this is convenient.